lib/mixlib/install/product.rb

```diff
@@ -181,45 +187,52 @@ def lookup(key, version = :latest)
         "chef-server-core"
       end
     end
-    ctl_command "chef-server-ctl"
-    config_file "/etc/opscode/chef-server.rb"
+    omnibus_project "chef-server"
+    ctl_command     "chef-server-ctl"
+    config_file     "/etc/opscode/chef-server.rb"
   end
 
   product "chef-server-ha-provisioning" do
-    product_name "Chef Server HA Provisioning for AWS"
-    package_name "chef-server-ha-provisioning"
+    product_name    "Chef Server HA Provisioning for AWS"
+    package_name    "chef-server-ha-provisioning"
+    omnibus_project "chef-server-ha-provisioning"
   end
 
   product "chefdk" do
-    product_name "Chef Development Kit"
-    package_name "chefdk"
+    product_name    "Chef Development Kit"
+    package_name    "chefdk"
+    omnibus_project "chefdk"
   end
 
   product "compliance" do
-    product_name "Chef Compliance"
-    package_name "chef-compliance"
-    ctl_command "chef-compliance-ctl"
-    config_file "/etc/chef-compliance/chef-compliance.rb"
+    product_name    "Chef Compliance"
+    package_name    "chef-compliance"
+    omnibus_project "chef-compliance"
+    ctl_command     "chef-compliance-ctl"
+    config_file     "/etc/chef-compliance/chef-compliance.rb"
   end
 
   product "delivery" do
-    product_name "Delivery"
-    package_name "delivery"
-    ctl_command "delivery-ctl"
-    config_file "/etc/delivery/delivery.rb"
+    product_name    "Delivery"
+    package_name    "delivery"
+    omnibus_project "delivery"
+    ctl_command     "delivery-ctl"
+    config_file     "/etc/delivery/delivery.rb"
   end
 
   product "ha" do
     product_name "Chef Server High Availability addon"
-    package_name "chef-ha"
-    config_file "/etc/opscode/chef-server.rb"
+    package_name    "chef-ha"
+    omnibus_project "chef-ha"
+    config_file     "/etc/opscode/chef-server.rb"
   end
 
   product "manage" do
     product_name "Management Console"
     package_name do |v|
       v < version_for("2.0.0") ? "opscode-manage" : "chef-manage"
     end
+    omnibus_project "manage"
```

Comment:
This should actually follow the same version logic and naming as the package name:
https://github.com/chef/chef-manage/blob/master/omnibus/config/projects/chef-manage.rb#L17
https://github.com/chef/omnibus-manage/blob/master/config/projects/opscode-manage.rb#L17